### ISSUE TITLE
Error when trying to pull plugin [Traefik]

### ISSUE THREAD

**Issue by zhivko0** [darkweak/souin#522]
Hi,
when I update the version number from v1.6.47 to v1.6.48/49 it gives me this error in the traefik log
`ERR Plugins are disabled because an error has occurred. error="unable to set up plugins environment: unable to download plugin github.com/darkweak/souin: failed to call service: Get \"https://plugins.traefik.io/public/download/github.com/darkweak/souin/v1.6.49\": GET https://plugins.traefik.io/public/download/github.com/darkweak/souin/v1.6.49 giving up after 4 attempt(s)"`
it works fine with versions older than 1.6.47. Sorry, I'm not exactly sure how the automation works, is this something that you control?

Thank you in advance!

**Comment by Shotster** [comment]
Same here!

**Comment by darkweak** [comment]
That's an error on their side, it works when I'm mounting the plugin from the source code to the path `/plugins-local/src/github.com/darkweak/souin` in træfik directly using `localPlugins` configuration key. Even if I try with a simple curl, it doesn't download the resource, but it exists because if I send a curl request on https://plugins.traefik.io/public/download/github.com/darkweak/souin/v1.6.50, it throws a 404 not found.

**Comment by zhivko0** [comment]
Thank you, so I guess local install is the option for now.

**Comment by AlphaCraft9658** [comment]
Are there any updates on the situation? As I am still experiencing the same issue.

**Comment by darkweak** [comment]
@AlphaCraft9658 that's an issue related to Træfik, not this repository.

**Comment by AlphaCraft9658** [comment]
> @AlphaCraft9658 that's an issue related to Træfik, not this repository.

Yeah, but I thought maybe someone knows something and I could just ask here instead of searching, but okay. I'm sorry if that's a problem.
